sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -220,22 +194,17 @@ def receive(self, batch=False, max_batch_size=300, max_wait_time=None):
             self._client._config.max_retries  # pylint:disable=protected-access
         )
         self._receive_start_time = self._receive_start_time or time.time()
-        deadline = self._receive_start_time + (max_wait_time or 0)  # max_wait_time can be None
+        deadline = self._receive_start_time + (
+            max_wait_time or 0
+        )
         if len(self._message_buffer) < max_batch_size:
-            # TODO: the retry here is a bit tricky as we are using low-level api from the amqp client.
-            #  Currently we create a new client with the latest received event's offset per retry.
-            #  Ideally we should reuse the same client reestablishing the connection/link with the latest offset.
```

Comment:
I actually don't know what this comment was referencing. I don't see a new client being created on retry, but I'm not 100% sure so will leave this in here for now? (Since everything is working, can we leave this in and investigate later? or should we investigate this before we GA?)